docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
 
-   @snippet snippets/Integrate_with_customer_application_new_API.cpp part10
+   @snippet snippets/src/main.cpp part9
 
    Make sure that shared input is kept valid during execution of each network. Otherwise, ROI blob may be corrupted if the original input blob (that ROI is cropped from) has already been rewritten.
 
-* Allocate input blobs of the appropriate types and sizes, feed an image and the input data to the blobs, and call `InferenceEngine::InferRequest::SetBlob()` to set these blobs for an infer request:
+* Allocate input blobs of the appropriate types and sizes, feed an image and the input data to the blobs, and call `ov::InferRequest::set_tensor()` to set these tensors for an infer request:
 
-   @snippet snippets/Integrate_with_customer_application_new_API.cpp part11
+   @snippet snippets/src/main.cpp part10
 
-A blob can be filled before and after `SetBlob()`.
+A blob can be filled before and after `set_tensor()`.
 
 > **NOTE**:
 >
-> * The `SetBlob()` method compares precision and layout of an input blob with the ones defined in step 3 and
+> * The `set_tensor()` method compares precision and layout of an input blob with the ones defined in step 3 and
 > throws an exception if they do not match. It also compares a size of the input blob with input
 > size of the read network. But if input was configured as resizable, you can set an input blob of
 > any size (for example, any ROI blob). Input resize will be invoked automatically using resize
 > algorithm configured on step 3. Similarly to the resize, color format conversions allow the color
 > format of an input blob to differ from the color format of the read network. Color format
 > conversion will be invoked automatically using color format configured on step 3.
 >
-> * `GetBlob()` logic is the same for pre-processable and not pre-processable input. Even if it is
+> * `get_tensor()` logic is the same for pre-processable and not pre-processable input. Even if it is
 > called with input configured as resizable or as having specific color format, a blob allocated by
 > an infer request is returned. Its size and color format are already consistent with the
 > corresponding values of the read network. No pre-processing will happen for this blob. If you
-> call `GetBlob()` after `SetBlob()`, you will get the blob you set in `SetBlob()`.
+> call `get_tensor()` after `set_tensor()`, you will get the blob you set in `set_tensor()`.
 
 #### Step 6. Start Inference
 
 Start inference in asynchronous or synchronous mode. Async API usage can improve overall frame-rate of the application, because rather than wait for inference to complete, the app can continue doing things on the host, while accelerator is busy.
```

Comment:
@myshevts  I suppose we agreed to demonstrate only async api? Or at least need to swap demonstrations: 1) async 2) sync